wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -175,4 +175,8 @@
 		}
 	} );
 
+	$( document ).ready(
+		alert("El Granto wuz here.");
```

Comment:
![I don't get it](http://www.quickmeme.com/img/5f/5f954e52b47fe0d1b9178990b178e2cad9c2c0b99255fa127affd74c8abb7761.jpg)